src/galileo/handlers/openai_agents.py

```diff
@@ -0,0 +1,333 @@
+import logging
+from datetime import datetime
+from typing import Any, Optional
+
+from agents import TracingProcessor, tracing
+
+from galileo.decorator import galileo_context
+from galileo.logger import GalileoLogger
+from galileo.utils.serialization import convert_to_string_dict, serialize_to_str
+
+_logger = logging.getLogger("galileo.handlers.agents")
```

Comment:
```suggestion
_logger = logging.getLogger(__name__)
```
similar as https://github.com/rungalileo/galileo-python/blob/19870ee044a2a76c0dffa5cfc1082f629b832643/src/galileo/jobs.py#L7